src/gui/src/displayControls.cpp

```diff
@@ -660,17 +660,70 @@ void DisplayControls::setDb(odb::dbDatabase* db)
 
   techInit();
 
+  double dbu_to_microns = tech->getDbUnitsPerMicron();
+
   for (dbTechLayer* layer : tech->getLayers()) {
     dbTechLayerType type = layer->getType();
     if (type == dbTechLayerType::ROUTING || type == dbTechLayerType::CUT) {
+      auto& row = layer_controls_[layer];
       makeLeafItem(
-          layer_controls_[layer],
+          row,
           QString::fromStdString(layer->getName()),
           layers_group_.name,
           Qt::Checked,
           true,
           color(layer),
           type == dbTechLayerType::CUT ? QVariant() : QVariant::fromValue(static_cast<void*>(layer)));
+      if (type == dbTechLayerType::ROUTING) {
+        auto units_convert = [](double value) -> QString {
+          int log_units = std::floor(std::log10(value) / 3.0) * 3;
+          if (log_units <= -18) {
+            return QString::number(value * 1e18) + " a";
+          } else if (log_units <= -15) {
+            return QString::number(value * 1e15) + " f";
+          } else if (log_units <= -12) {
+            return QString::number(value * 1e12) + " p";
+          } else if (log_units <= -9) {
+            return QString::number(value * 1e9) + " n";
+          } else if (log_units <= -6) {
+            return QString::number(value * 1e6) + " \u03BC";
```

Comment:
const char* micron = "\u03BC";
QString::number(value * 1e6) + " " + micron;

is more readable and micron can be used further down as well.